serving/src/main/java/ai/djl/serving/pyclient/PythonConnector.java

```diff
@@ -13,53 +13,55 @@
 package ai.djl.serving.pyclient;
 
 import ai.djl.serving.pyclient.protocol.ResponseDecoder;
+import ai.djl.serving.util.ConfigManager;
+import ai.djl.serving.util.Connector;
 import io.netty.bootstrap.Bootstrap;
 import io.netty.channel.Channel;
 import io.netty.channel.ChannelFuture;
 import io.netty.channel.ChannelInitializer;
 import io.netty.channel.EventLoopGroup;
-import io.netty.channel.nio.NioEventLoopGroup;
-import io.netty.channel.socket.SocketChannel;
-import io.netty.channel.socket.nio.NioSocketChannel;
+import io.netty.channel.unix.DomainSocketAddress;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /** This class creates a netty client. */
-public class SocketConnector {
+public class PythonConnector {
 
-    private static final Logger logger = LoggerFactory.getLogger(SocketConnector.class);
-    private static final SocketConnector SOCKET_CONNECTOR = newInstance();
+    private static final Logger logger = LoggerFactory.getLogger(PythonConnector.class);
+    private static final PythonConnector PYTHON_CONNECTOR = newInstance();
+    private static final String DEFAULT_SOCKET_PATH = "/tmp/uds_sock";
```

Comment:
Is this okay, to hardcode the default socket path or shall we get it from config?